### PR TITLE
General | Refresh every APP state after save event

### PR DESCRIPTION
**Pull request by szymon-owczarzak**
## Description

More details in related issue: https://github.com/wttech/cogboard/issues/346

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](https://github.com/wttech/cogboard/blob/master/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] Automated functional tests have been added or modified to cover my changes (if applicable)
- [ ] I have updated the documentation accordingly.

---

I hereby agree to the terms of the Cogboard Contributor License Agreement.

### CODE REVIEW

File: cogboard-webapp/src/constants/index.js [hunk 1]
```diff
@@ -331,5 +331,10 @@ export const NOTIFICATIONS = {
     type: 'error',
     message: message,
     duration: 3000
+  }),
+  CONFIG_LOADED: () => ({
+    type: 'info',
+    message: 'Config Loaded',
+    duration: 3000
```
Comment (szymon-owczarzak) [inline cogboard-webapp/src/constants/index.js:338]: Cogboard will now display a message each time a new config is loaded in the background.

File: cogboard-webapp/src/actions/actionCreators.js [hunk 2]
```diff
@@ -129,10 +133,6 @@ export const dataChanged = () => ({
   type: DATA_CHANGED
 });
 
-export const saveDataStart = () => ({
-  type: SAVE_DATA_START
-});
```
Comment (szymon-owczarzak) [inline cogboard-webapp/src/actions/actionCreators.js:134]: not used

File: cogboard-webapp/src/App.js [hunk 3]
```diff
@@ -58,8 +66,9 @@ function App() {
 
         if (eventType === 'widget-update') {
           dispatch(updateWidgetContent(data));
-        } else if (eventType === 'notification-config-save') {
+        } else if (eventType === 'config-saved') {
           dispatch(saveDataSuccess());
+          dispatch(refetchInitData());
```
Comment (szymon-owczarzak) [inline cogboard-webapp/src/App.js:71]: on SAVE event success - fetch config again

**Review by devzbysiu — APPROVED** [review]

File: cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/controller/ConfirmationSender.kt [hunk 4]
```diff
@@ -9,26 +11,23 @@ import io.vertx.core.logging.LoggerFactory
 class ConfirmationSender(private val vertx: Vertx) {
 
     fun sendOk() {
-        vertx.eventBus().send(CogboardConstants.EVENT_SEND_MESSAGE_TO_WEBSOCKET,
-                JsonObject().message(OK_MESSAGE))
+        vertx.eventBus().send(Event.SEND_MESSAGE_TO_WEBSOCKET, JsonObject().message(OK_MESSAGE))
     }
 
     fun sendError(body: JsonObject) {
```
Comment (yahorm) [inline cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/controller/ConfirmationSender.kt:17]: Is this method used somewhere?

**Review by yahorm — DISMISSED** [review]
great job @szymon-owczarzak! the constants have become more readable :)

File: cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/controller/ConfirmationSender.kt (hunk 4 above)
Comment (szymon-owczarzak) [reply cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/controller/ConfirmationSender.kt:17]: No it is not i will delete

**Pull request merged**